src/model/mapping/mapping_model.py

```
@@ -58,22 +60,28 @@ def get_nomenclatures_groups(
+        # Predict groups ids and encode them to groups names
+        predicted_groups = label_encoder.inverse_transform(
+            model.predict(prediction_list),
+        )
 
-    return predicted_groups
+        return predicted_groups
+    except ValueError:
+        raise Exception(
+            f"Model with ID '{model_id}' does not support params or DataFrame for prediction does "
+            f"not contains them"
+        )
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)